examples/two_neuron_example.py

```diff
@@ -24,57 +27,53 @@
 # === Define parameters ========================================================
 
 cell_params = {
-    "v_rest": -70.0, # (mV)
-    "v_reset": -70.0, # (mV)
-    "cm": 0.250, # (nF)
-    "tau_m": 10, # (ms)
-    "tau_refrac": 2, # (ms)
-    "tau_syn_E": 2, # (ms)
-    "tau_syn_I": 2, # (ms)
-    "v_thresh": -55.0, # (mV)
-    "i_offset": 0, # (nA)
+    "v_rest": -70.0,  # (mV)
+    "v_reset": -70.0,  # (mV)
+    "cm": 0.250,  # (nF)
+    "tau_m": 10,  # (ms)
+    "tau_refrac": 2,  # (ms)
+    "tau_syn_E": 2,  # (ms)
+    "tau_syn_I": 2,  # (ms)
+    "v_thresh": -55.0,  # (mV)
+    "i_offset": 0,  # (nA)
 }
 
 # === Build the network ========================================================
 
-sim.setup(timestep=0.1) # (ms)
+sim.setup(timestep=0.1)  # (ms)
 
 cell_type = sim.IF_curr_alpha(**cell_params)
-neuron1 = sim.Population(1, cell_type, label="Neuron 1")
-neuron1.set(i_offset=0.376)
-neuron1.record("v")
-
-neuron2 = sim.Population(1, cell_type, label="Neuron 2")
-neuron2.record("v")
+neurons = sim.Population(2, cell_type, initial_values={"v": -70}, label="Neurons")
+neurons[0:1].set(i_offset=0.376)
+neurons.record("v")
 
 syn = sim.StaticSynapse(weight=0.02, delay=1.0)
-projection1 = sim.Projection(neuron1, neuron2, sim.AllToAllConnector(), syn)
+projection1 = sim.Projection(neurons[0:1], neurons[1:2], sim.AllToAllConnector(), syn)
 
 # === Run simulation ===========================================================
 
 sim.run(1000)
 
-neuron1_data_v = neuron1.get_data().segments[0].filter(name="v")[0]
-neuron2_data_v = neuron2.get_data().segments[0].filter(name="v")[0]
+data = neurons.get_data().segments[0].filter(name="v")[0]
```

Comment:
I will check it on my end but still to confirm, does "data" variable would contain an array of 2 neurons with their respective data?

If not, then how can we record the data for both neurons?

Reply:
Yes, it contains an array with shape (n, 2)  (where n is the number of time points).